rkt/gc.go

```diff
@@ -238,14 +193,22 @@ func deletePod(p *pod) {
 		}
 		defer s.Close()
 
-		if err := cleanExitedGarbage(p, s); err != nil {
-			stderr("Received error performing GC for pod %q: %v", p.uuid, err)
-			stderr("Attempting forced GC of pod %q...", p.uuid)
-			if err := stage0.ForceMountGC(p.path(), p.uuid.String()); err != nil {
-				stderr("Forced GC of pod %q failed: %v\n", p.uuid, err)
-				return
-			}
-			stderr("Forced GC of pod %q succeeded.", p.uuid)
+		// execute stage1's GC
+		stage1TreeStoreID, err := p.getStage1TreeStoreID()
+		if err != nil {
+			stderr("error getting stage1 treeStoreID: %v", err)
+			return
```

Comment:
Yeah, if we can't get the `treeStoreID` we should still be able to force-GC the pod.